tests/models/test_roberta.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from fms.testing._internal.model_test_suite import (
+    ConfigFixtureMixin,
+    ModelFixtureMixin,
+    ModelConfigTestSuite,
+    ModelConsistencyTestSuite,
+    ModelCompileTestSuite,
+)
+from fms.models.roberta import RoBERTaConfig, RoBERTa
+
+
+class RoBERTaFixtures(ConfigFixtureMixin, ModelFixtureMixin):
+    """
+    Base RoBERTa Fixtures that can be re-used for other purposes
+
+    This will include the config and model signatures
+    """
+
+    @pytest.fixture(scope="class", autouse=True)
+    def uninitialized_model(self, config: RoBERTaConfig):
+        return RoBERTa(config)
+
+    @pytest.fixture(scope="class", autouse=True)
+    def config(self) -> RoBERTaConfig:
+        return RoBERTaConfig(
```

Comment:
would it make sense to use `_micro_char_config` in this kind of test?

Reply:
We could, however I believe the Hf tests are using some other tokenizer by default (so I set it to that size), that can be changed though most likely with no issue. I would want to do something similar for all models as well though.